install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,6 +40,14 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: ClusterVersionOperatorNotRetrieving
+      annotations:
+        message: Cluster version operator has not retrieved updates for 10 mins. Failure reason {{ "{{ $labels.reason }}" }}. {{ "{{ $labels.message }}" }}.
```

Comment:
nit: "mins" -> "minutes".  Less mental overhead from guessing at abbreviations.

Reply:
I will change them all in CVO. I thought that perhaps there was an OpenShift wide standard for Alert text and that mins was part of that but apparently not. Foolish me...

Reply:
hah, I dunno, maybe there is and I'm not aware of it.  If there's an existing pattern you're matching (sounds like there is), feel free to punt, and I'll take this on in follow-up work